xarray/tests/__init__.py

```diff
@@ -53,6 +53,13 @@ def _importorskip(modname, minversion=None):
     return has, func
 
 
+class LooseVersion(version.LooseVersion):
+    def __init__(self, vstring=None):
+        if '+' in vstring:
+            vstring = vstring[:vstring.find('+')]
+        super(LooseVersion, self).__init__(vstring)
```

Comment:
Rather than a subclass, can we just make an alternate constructor? I think that would be less error prone, e.g.,
```python
def LooseVersion(vstring):
    if '+' in vstring:
        vstring = vstring[:vstring.find('+')]
    return distutils.LooseVersion(vstring)
```

Reply:
(if you're in the market for much more refine-y comments, you could use `vstring.split('+')[0]`, which works whether or not there's a `+`)

Reply:
Thanks. Updated